cmd/webhook/main.go

```diff
@@ -124,6 +131,10 @@ func main() {
 			webhook.MutateHandler(w, r)
 		})
 
+		http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
can you add a deployment example with health check (with liveness probe) in https://github.com/k8snetworkplumbingwg/network-resources-injector/blob/master/deployments/server.yaml ?


as in k8s liveness http probe example:
```
apiVersion: v1

kind: Pod

metadata:

  labels:

    test: liveness

  name: liveness-http

spec:

  containers:

  - name: liveness

    image: k8s.gcr.io/liveness

    args:

    - /server

    livenessProbe:

      httpGet:

        path: /healthz

        port: 8080

        httpHeaders:

        - name: Custom-Header

          value: Awesome

      initialDelaySeconds: 3

      periodSeconds: 3

```